test/protected_resource_test.rb

```diff
@@ -0,0 +1,90 @@
+require 'test_helper'
+
+class TestResource < Minitest::Test
+  SomeError = Class.new(StandardError)
+
+  def setup
+    Semian.destroy(:testing)
+  rescue
+    nil
+  end
+
+  def teardown
+    destroy_resources
+  end
+
+  def test_acquire_without_bulkhead
```

Comment:
Yeah, this test doesn't seem to do this. I'd suggest trying to make this be acquired from, say, 10 processes in parallel to make sure bulkheads are _not_ enabled.